src/apis/vaults.ts

```diff
@@ -209,6 +212,20 @@ export class DefaultVaultsAPI {
         return new UInt(new TypeRegistry(), 0) as PolkaBTC;
     }
 
+    private async getSecureCollateralThreshold(): Promise<u128> {
+        return await this.api.query.vaultRegistry.secureCollateralThreshold();
+    }
+
+    async getIssuablePolkaBTC(): Promise<string> {
+        const collateral = new DefaultCollateralAPI(this.api);
+        const totalLockedDot = await collateral.totalLockedDOT();
```

Comment:
Under the hood, this will return the total locked dot denoted in planck. You need to convert this into dot before doing the calculation by multiplying with DOT_IN_PLANCK constant from utils.